### ISSUE TITLE
Dubious 2h bid. When partner has passed to opening, bidding a 2nd time should be constructive

### ISSUE THREAD

**Issue by ThePokerDude** [EdwardPiwowar/BBA#283]
I think east should pass after 2c. 
Partner has a max of 6hCP.
The heart suit is weak.
We have absolute minimum.
The opponent who doubled often has 4h.
0F2D6E5E2CFB4BF6D7483E411598
![image](https://github.com/EdwardPiwowar/BBA/assets/33804161/db646a22-4528-4e81-8a33-ab3875539ed9)

**Comment by ThorvaldAagaard** [comment]
I think we have seen this theme quite often. opener should not bid unless distribution or extra.

**Comment by EdwardPiwowar** [comment]
Ok, will be changed:
12+ or 13+ for red.
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/7269df8c-5481-4e99-a66d-eac2bb6da661)

**Comment by ThorvaldAagaard** [comment]
With 5431 opposite a passed partner I think you should have 5-5 if only 12 hcp

**Comment by EdwardPiwowar** [comment]
I will leave it like that for now because my tests do not show any significant differences in effectiveness.

**Comment by ThorvaldAagaard** [comment]
Remember we are not trying to find new and better way (yet) to bid, but trying to describe the most common way of playing, so the robots can play with humans. Bidding 2X opposite a passed partner is beginners mistake, but will normally not be punished before you meet good opponents, 
So yes, the simulations will not show differences because the opponents will not take advantage, but it doesn't make the bid more correct.

**Comment by EdwardPiwowar** [comment]
That's why I raised the limit. Do you think too less?

**Comment by ThorvaldAagaard** [comment]
Yes, HCP + 2 longest suits should be 23 and perhaps even 24 when red.

**Comment by ThePokerDude** [comment]
Fixed in 8627
![image](https://github.com/user-attachments/assets/b5838d68-13c5-45a9-9ed4-a8c8b514342f)

**Comment by ThePokerDude** [comment]
All versions from 8643 on stop in 3c instead of 3nt
![image](https://github.com/user-attachments/assets/cb6a8c51-5ba9-4011-a52a-593ea7b45217)

**Comment by ThorvaldAagaard** [comment]
I have created a script, that reads all the issues from github, and extract the BBA-id for any board mentioned. Perhaps it is interesting to run regression test on those, to see if any has changed.

**Comment by ThePokerDude** [comment]
Yes, definitely. I am currently using a file you created this way. It contains 601 issues.
I see alot of changes from version to version.

**Comment by ThorvaldAagaard** [comment]
Here is a csv-file with all issues

[github_issues.csv](https://github.com/user-attachments/files/17612505/github_issues.csv)

I might have created a regex to find all deals, but can't find it, so let me know if you want any postprocssing of the csv-file
